RELEASING.md

```diff
@@ -26,3 +26,54 @@
         helm repo update elastic
         helm search repo -l elastic/apm-attacher
         ```
+
+## Testing procedure (for Linux, Windows works too but you need to adjust how files are created or create them manually)
+
```

Comment:
I guess there is a prereq assumption that one has a k8s cluster to work with.

Reply:
You think that needs to be explicit?

Reply:
Nah, it is fine.